etc/docker-scripts/bootstrapper.py

```diff
@@ -13,19 +13,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import shutil
 import subprocess
 import sys
+from packaging import version
 
 
-def import_with_auto_install(package):
-    try:
-        return __import__(package)
-    except ImportError:
-        print('Updating package {}'.format(package))
-        subprocess.check_call([sys.executable, '-m', 'pip', 'install', '--upgrade', package])
+def package_install():
+
+    elyra_package_list = package_list_to_dict("requirements-elyra.txt")
+    current_package_list = package_list_to_dict("requirements-current.txt")
+    to_install_list = []
+
+    for package, ver in elyra_package_list.items():
+        if package in current_package_list.keys():
+            if version.parse(ver) > version.parse(current_package_list[package]):
+                print("Updating %s package from version %s to %s" % (package, current_package_list[package], ver))
```

Comment:
Could we add ellipses to the `Updating` and `Installing` statements so they're consistent with `Skipping...`?  Or add/change them to periods.